test/schema.spec.ts

```diff
@@ -18,7 +20,44 @@ describe.each([
   { schemaType: 'alias', schema: schemas.alias, example: alias }
 ])(`Run validate for all schemas`, ({ schemaType, schema, example }) => {
   test(`validating schema ${schemaType} should return true`, () => {
-    const isValid = validateSchema(schema, example, { snapshotEnv: 'mainnet' });
+    const isValid = validateSchema(schema, example, {
+      snapshotEnv: 'mainnet'
+    });
     expect(isValid).toBe(true);
   });
 });
+
+// Tests for turbo spaces
+
+describe.each([
+  { schemaType: 'space', schema: schemas.space, example: space },
+  { schemaType: 'proposal', schema: schemas.proposal, example: proposalTurbo }
+])(`Run validate for turbo spaces`, ({ schemaType, schema, example }) => {
+  test(`validating schema ${schemaType} should return true`, () => {
+    const isValid = validateSchema(schema, example, {
+      snapshotEnv: 'mainnet',
+      spaceType: 'turbo'
+    });
+    expect(isValid).toBe(true);
+  });
+});
+
+describe.each([
+  // { schemaType: 'space', schema: schemas.space, example: space },
+  {
+    schemaType: 'proposal',
+    schema: schemas.proposal,
+    example: proposalTurbo,
```

Comment:
This file name is confusing, as it's named turbo, but will not pass any turbo test

Reply:
Sorry I don't get what you mean 🙈 

Here we test if turbo payload returns error.